docs/application/native/guides/multimedia/image-recognition.md

```diff
@@ -310,30 +324,30 @@ To track images:
 
    1. Use the `mv_image_track()` function to track the target image object in the preview images:
 
-    ```
-    error_code = mv_image_track(imagedata.g_source, imagedata.g_image_track_model,
-                                imagedata.g_engine_config, _on_image_tracked_cb, NULL);
-    if (error_code != MEDIA_VISION_ERROR_NONE)
-        dlog_print(DLOG_ERROR, LOG_TAG, "error code= %d", error_code);
-    ```
+      ```
+      error_code = mv_image_track(imagedata.g_source, imagedata.g_image_track_model,
+                                  imagedata.g_engine_config, _on_image_tracked_cb, NULL);
+      if (error_code != MEDIA_VISION_ERROR_NONE)
+          dlog_print(DLOG_ERROR, LOG_TAG, "error code= %d", error_code);
+      ```
 
    2. The `mv_image_track()` function invokes the `_on_image_tracked_cb()` callback.The following callback example prints the location of the target image object.
```

Comment:
line 334: The separated lines are merged as one line.

Reply:
done